test/client.js

```diff
@@ -432,6 +432,17 @@ describe('Rest Client', function () {
 
       client.get(url, done);
     });
+
+    it('should pass the response object through', function (done) {
```

Comment:
Can you update these to be say `passes the response object through` without the `should`? Just to match the styles of the other tests.